tests/integration/setup_test.go

```diff
@@ -58,7 +57,12 @@ var _ = BeforeSuite(func() {
 	ebs, err = cloud.NewCloud()
 	Expect(err).To(BeNil(), "Set up Cloud client failed with error")
 	drv = driver.NewDriver(ebs, nil, endpoint)
-	go drv.Run()
+	go func() {
+		err := drv.Run()
+		if err != nil {
+			fmt.Println("Driver exits: ", err)
```

Comment:
Can we use something like: `log.Fatalf("Failed to start CSI driver: %v, err")`

Reply:
Since gomega `Expect` is used in integration test, I changed to use `Expect(err).To(BeNil())`. Let me know if you think `log` is needed